zbus/Cargo.toml

```diff
@@ -25,7 +25,7 @@ camino = ["zvariant/camino"]
 # Enables API that is only needed for bus implementations (enables `p2p`).
 bus-impl = ["p2p"]
 # Enables API that is only needed for peer-to-peer (p2p) connections.
-p2p = ["dep:rand"]
+p2p = ["uuid/v4"]
```

Comment:
hmm.. actually we could still keep `uuid` optional but required by `p2p` feature and always enable `v4` feature of `uuid` in zbus (only, zvariant shouldn't need to require `v4` I think).

Reply:
> we could, but then that would mean I can't use uuid's `try_parse()` in `validate_guid()` 🫤

I'm sorry, I misunderstood and thought that `validdate_guid` is needed for `p2p` only. Let's keep this as is.